### YamlMime:UniversalReference
ms.openlocfilehash: f60364c788f18b518c95ea9fd292bde5921b37b0
ms.sourcegitcommit: efa2d98deffe8a0d41a8d63f9f07aa720862e6ab
ms.translationtype: MT
ms.contentlocale: zh-CN
ms.lasthandoff: 12/13/2018
ms.locfileid: "52005441"
items:
- uid: azure-iot-common.ArgumentOutOfRangeError
  name: ArgumentOutOfRangeError
  fullName: ArgumentOutOfRangeError
  children:
  - azure-iot-common.ArgumentOutOfRangeError.constructor
  - azure-iot-common.ArgumentOutOfRangeError.Error
  langs:
  - typeScript
  type: class
  summary: 自变量具有一个值，超出了允许的范围时引发的错误。
  extends:
    name: Error
  package: azure-iot-common
- uid: azure-iot-common.ArgumentOutOfRangeError.constructor
  name: ArgumentOutOfRangeError(string)
  children: []
  type: constructor
  langs:
  - typeScript
  summary: ''
  syntax:
    content: 'new ArgumentOutOfRangeError(message?: string)'
    parameters:
    - id: message
      type:
      - string
      description: ''
      optional: true
  package: azure-iot-common
- uid: azure-iot-common.ArgumentOutOfRangeError.Error
  name: Error
  fullName: Error
  children: []
  langs:
  - typeScript
  type: property
  summary: ''
  syntax:
    content: 'static Error: ErrorConstructor'
    return:
      type:
      - ErrorConstructor
  package: azure-iot-common
